src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -1,15 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;net5.0</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultItems>false</EnableDefaultItems>
     <IsPackable>true</IsPackable>
     <DeviceRoot>$(MSBuildThisFileDirectory)../devices/</DeviceRoot>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage)</TargetsForTfmSpecificContentInPackage>
-    <LangVersion>8</LangVersion>
+    <LangVersion>9</LangVersion>
```

Comment:
preview is fine with me as well

Reply:
I'm good with either. I personally think that the integer value is better if it is correctly descriptive. We can always moved to preview if we start using newer features. That won't be for a while since C# 10 won't be usable for some time.